### DISCUSSION TITLE
How to ask caching ignore certain files?

### DISCUSSION THREAD

**Discussion by michael-land** [vercel/turborepo#781] [Help] [answered]
My use case is I don't want invalid cache when editing README file.

```
# editing these should not rebuild
apps/app1/README.md
apps/app2/README.md
```

I only find this, but that's for the opposite purpose

https://turborepo.org/docs/features/caching#alter-caching-based-on-environment-variables-and-files

**Comment by samchouse** [comment]
Try this https://turborepo.org/docs/reference/command-line-reference#--ignore

**Comment by anthonyshew** [comment, marked as answer]
The `inputs` key can be used to ignore certain globs. Ignore READMEs could be done with:

```
{
  "pipeline": {
    "build": {
      // ...
      "inputs": ["!README.md"],
    },
  }
}

```
